### ISSUE TITLE
[macOS] Immediate crash

### ISSUE THREAD

**Issue by firelad97** [jpd002/Play-#777]
OS: macOS Mojave 10.14.5 with Metal
Play! version: 05-July-2019

Previously, I was able to run Play! emulator just fine. It loaded games just fine as well. Suddenly, today it decided to crash. I tried uninstalling and re-installing, the crash still persist. No GUI showing up.

[Here is the log that occurred after the crash, the "problem report".](https://gist.github.com/firelad97/6628f9d677f0d8b033575a8281929be8)

This repo seem to have no issue template, so if you need more information, please let me know. Thanks!

**Comment by Zer0xFF** [comment]
the crash seems to related to a gamepad of sorts, do you have 1 plugged in? if not, did you plug a new device into your Mac recently?

**Comment by firelad97** [comment]
Yes to both. Steam controller.

**Comment by Zer0xFF** [comment]
this isn't a solution, but try running play without connecting the controller, and see if it crashes, if plugged in while Play! is running

**Comment by firelad97** [comment]
I've tested this previously, Play! does not crash while controller is not plugged in.

However, of all days, today Play! no longer crash with controller plugged in when starting it. I'm not sure how I did do it, but I think it was because I had restarted computer, start Play! and then plug in controller, close Play! and re-open it while controller is still plugged in. I don't know if this issue will occur again later.
